### PR TITLE
Lesson 1 7

### PR DESCRIPTION
**Pull request by ozzydogandev**
- Asynchronous data fetching added
- Loading state added

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,32 +3,54 @@ import './App.css';
...
+  }, [todoList, isLoading]);
+
+  const addTodo = (newTodo) => setTodoList([...todoList, newTodo]);
+
+  const removeTodo = (id) => {
+    const updatedTodoList = todoList.filter((todo) => todo.id !== id);
+    setTodoList(updatedTodoList);
+  };
+
+  if (isLoading) {
```
Comment (AnnaPestova1) [inline src/App.jsx]: Hi! can you please switch if-else to the ternary operator, to complete the assignment? 
image

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Oguzhan! You did a great work and your code works as expected. 

I am requested only one change - to change if-else statement to the ternary operator as it requested by assignment. It is good to know, that if we, as developers have particular documentation or instructions, we should follow them. If not - we can choose our own approach. We have instruction about ternary operator. 

When you'll make changes, please, fill [this form](https://airtable.com/appoSRJMlXH9KvE6w/shrBpqHbS6wgInoF9?prefill_Lessons=React%20v2%3A%20Lesson%207%20-%20Lesson%201.7%3A%20Asynchronous%20Data%20Conditional%20Rendering%20Advanced%20and%20Impossible%20State) again and I will review your assignment ASAP

I think this is good article about why we prefer use ternary operator in jsx https://www.dhiwise.com/post/how-to-use-the-ternary-operator-in-react-for-cleaner-code

File: src/App.jsx (hunk 1 above)
Comment (ozzydogandev) [reply src/App.jsx]: Hi Anna! Thanks for checking that. I fixed the code and used ternary operator instead. I appreciate your mentorship🙏🏻

**Review by AnnaPestova1 — APPROVED** [review]
Great work! Thank you for changes

**Pull request merged**